app/dashboard/views/unsubscribe.py

```diff
@@ -68,3 +70,46 @@ def block_contact(contact_id):
         )
     else:  # ask user confirmation
         return render_template("dashboard/block_contact.html", contact=contact)
+
+
+@dashboard_bp.route("/encoded", methods=["GET"])
+@login_required
+def encoded_unsubscribe():
```

Comment:
can we add a small doc on what to expect as input for this route?

Reply:
Will do.

Reply:
Done